docs/csharp/tutorials/index.md

```diff
@@ -14,14 +14,14 @@ ms.assetid: fcc83b5b-fb68-4e48-9132-0882677d8056
 
 # C# Tutorials
 
-These exercises enable you to build C# programs using core CLR.
+These tutorials enable you to build C# programs using [CoreCLR](https://github.com/dotnet/coreclr).
 
-* [Console Application](console-teleprompter.md). This tutorial
+* [Console Application](console-teleprompter.md): This tutorial
```

Comment:
Since we're already saying each of these are tutorials in the introduction, maybe we could cut the first part of the sentence and say:
[Console Application](console-teleprompter.md): demonstrates Console I/O, ....
It sounds less repetitive to me.